src/pages/podcasts/[slug].tsx

```diff
@@ -4,6 +4,7 @@ import {loadPodcast, loadPodcasts} from 'lib/podcasts'
 import Rss from '../../components/images/rss.svg'
 import Spotify from '../../components/images/spotify.svg'
 import ApplePodcasts from '../../components/images/apple-podcasts.svg'
+import PodcastCard from '../../components/podcasts/card'
```

Comment:
you can use `components/podcasts/card` here as all top level `src/*` folders are aliased.

Reply:
Rad 😎 🚀